libs/service-portal/wip/README.md

```diff
@@ -0,0 +1,7 @@
+# Service Portal Wip
```

Comment:
consider putting some information about the library here

Reply:
Absolutely!